src/api/integrations/managed-integrations.md

```diff
@@ -0,0 +1,123 @@
+---
+layout:  default
+grand_parent: API Reference
+parent: Integrations
+title: Managed Integrations
+permalink: /api/managed-integrations
+---
+
+# Managed Integration
+{:.no_toc}
+
+## Contents
+{:.no_toc .text-delta}
+
+* TOC
+{:toc}
+
+## Models
+
+### Managed Integration
+
+{% h4 Mandatory Fields %}
+
+|       Field        |        Type        |                                                  Description                                                   |
+| :----------------- | :----------------- | :------------------------------------------------------------------------------------------------------------- |
+| id                 | String             | The Managed Integration's unique identifier.                                                                   |
+| type               | String             | The type of Managed Integration. Supported types are `kete-enrolment`.                                         |
```

Comment:
If we're exposing the possible types can we add a table for them

Reply:
I have addressed this.

Reply:
Is there a reason for this? I don't think this is currently necessary.

Reply:
I'm thinking It won't be long until more types are here seeing it's needed by a few projects. I'd say it's good to get most of the work done whilst you're working on it to reduce rework. But if you think it should be done later then that's fine too.